### PR TITLE
Homework7 8

### PR DESCRIPTION
**Pull request by mikdikiy**

### CODE REVIEW

File: homework7_8/style.css [hunk 1]
```diff
@@ -0,0 +1,701 @@
...
+ul{list-style: none;}
+.container{flex-grow: 2;background-color: var(--back-container-color);}
+.footer-content{flex-grow: 1;}
+.header,.slider-viewport,.slider-pager, .about-content,
+.content, .logos-wrapper, .footer-comments,.footer-menu-wrapper
+{
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+@media (min-width: 960px){
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: all media breakpoints should be in constants

File: homework7_8/style.css [hunk 2]
```diff
@@ -0,0 +1,701 @@
...
+  .header,.slider-viewport,.slider-pager, .about-content,
+  .content, .logos-wrapper, .footer-comments,.footer-menu-wrapper{max-width: 960px;} 
+}
+@media (min-width: 768px) and (max-width: 959.99px){
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: don't use 2 breakpoints in same time, next rules should modify your current rules

File: homework7_8/style.css [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+.footer-content{flex-grow: 1;}
+.header,.slider-viewport,.slider-pager, .about-content,
+.content, .logos-wrapper, .footer-comments,.footer-menu-wrapper
+{
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+@media (min-width: 960px){
+  .header,.slider-viewport,.slider-pager, .about-content,
+  .content, .logos-wrapper, .footer-comments,.footer-menu-wrapper{max-width: 960px;}
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: a lot of classes for same, use wrapper class for this reasons

File: homework7_8/style.css [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+  top:106px; left: 50%; margin-right: -50%;
+  transform: translate(-50%, 3px)
+  }
+  .menu-chk{
+    width: 30px;
+    cursor: pointer;
+    position: relative;
+    z-index: 23;
+    top:25px;
+    margin-top: -17px;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: avoid margin-top

File: homework7_8/style.css [hunk 5]
```diff
@@ -0,0 +1,701 @@
...
+@media (min-width: 768px){.break-point-width{max-width: 728px;}}
+@media (min-width: 960px){.break-point-width{max-width: 960px;}}
+
+/* -----header and slider----- */
+.header-and-slider{
+  background-color: var(--back-footer-and-slider-color);
+  background-image: url(./img/Background.jpg);
+  background-repeat: no-repeat;
+  background-position:top;
+  background-repeat: repeat-x;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css:78]: duplicated background-repeat

File: homework7_8/style.css [hunk 6]
```diff
@@ -0,0 +1,701 @@
...
+    gap:35px;
+    padding-left: 0;
+    margin-right: auto;   
+    flex-direction: row;
+    width: auto;
+    overflow-x: auto;
+    border: none; border-radius: none;
+    background-color: rgba(0,0,0,0);
+    position: static;
+    top: none; left: none; margin-right: none;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: duplicated margin-right, please use editor align function to follow code style conventions, one property for line

File: homework7_8/style.css [hunk 7]
```diff
@@ -0,0 +1,701 @@
...
+  .header-menu{
+    display: flex;
+    justify-content: space-between;
+    gap:35px;
+    padding-left: 0;
+    margin-right: auto;   
+    flex-direction: row;
+    width: auto;
+    overflow-x: auto;
+    border: none; border-radius: none;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: border-radius: Mismatched property value (<length-percentage>{1,4} [/ <length-percentage>{1,4}]? | [initial | inherit | unset | revert])

File: homework7_8/style.css (hunk 6 above)
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: top, left, margin: Mismatched property value ([<length> | <percentage> | fill | auto] | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 8]
```diff
@@ -0,0 +1,701 @@
...
+  .header{
+    display: flex; flex-direction: row;
+    padding: 49px 0 156px 0} 
+}
+.slider-overlay{
+  min-height: 400px;
+  background-color: var(--slider-overlay-color);
+  display: flex;
+}
+.slider-viewport{overflow-x: clip;}
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: Mismatched property value ([visible | hidden | scroll | auto | paged-x | paged-y | paged-x-controls | paged-y-controls | fragments | -webkit-marquee | -moz-scrollbars-none | -moz-scrollbars-horizontal | -moz-scrollbars-vertical | -moz-hidden-unscrollable] | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 9]
```diff
@@ -0,0 +1,701 @@
...
+}
+.about-wrapper{
+  display: flex; flex-direction: column;
+}
+.about-wrapper-heading{
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  margin-bottom: 0;
+  font-weight: 500;
+  margin:0; padding: 5px;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: duplicate margin property

File: homework7_8/style.css [hunk 10]
```diff
@@ -0,0 +1,701 @@
...
+}
+
+/* -----content----- */
+.content{
+  padding: 3% 5px 1% 0;
+  display: flex;
+}
+.testimonial-and-updates{
+  display: flex;
+  align-items: top;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: Mismatched property value (flex-start | flex-end | center | baseline | stretch | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 11]
```diff
@@ -0,0 +1,701 @@
...
+  justify-content: space-between;
+}
+.testimonial-layout{margin-right: 5px; padding-bottom: 100px;}
+.testimonials-cut-wrapper{
+  display:flex;
+  overflow-x: clip;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css:552]: Mismatched property value ([visible | hidden | scroll | auto | paged-x | paged-y | paged-x-controls | paged-y-controls | fragments | -webkit-marquee | -moz-scrollbars-none | -moz-scrollbars-horizontal | -moz-scrollbars-vertical | -moz-hidden-unscrollable] | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 12]
```diff
@@ -0,0 +1,701 @@
...
+.updates-post{align-items: flex-start;}
+.logos-wrapper{-flex-wrap: wrap; row-gap: 20px; padding-left: 1%;}
+@media (min-width: 480px){  
+  .testimonials{max-width: 417px;}
+  .testimonials-cut-wrapper{max-width: 443px;}
+  .testimonial-heading{
+    font-size: 30px;
+    flex-direction: row;
+    line-height: 1;
+    align-items: left;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: Mismatched property value (flex-start | flex-end | center | baseline | stretch | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 13]
```diff
@@ -0,0 +1,701 @@
...
+}
+@media (min-width: 768px){
+  .testimonial-and-updates{flex-direction: row;}
+  .testimonials{max-width: 313px;}
+  .testimonials-cut-wrapper{max-width: 339px;}
+  .testimonial-heading{
+    font-size: 24px;
+    flex-direction: column;
+    line-height: 1;
+    align-items: left;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: Mismatched property value (flex-start | flex-end | center | baseline | stretch | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 14]
```diff
@@ -0,0 +1,701 @@
...
+    align-items: center;
+  }
+}
+
+/* -----footer----- */
+.footer-content{background-color: var(--footer-back-color);}
+.footer-comments{
+  display: flex;
+  justify-content: space-between;
+  align-items: top;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css]: Mismatched property value (flex-start | flex-end | center | baseline | stretch | [initial | inherit | unset | revert])

File: homework7_8/style.css [hunk 15]
```diff
@@ -0,0 +1,701 @@
...
+}
+.comment-text{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px; font-weight: 500;
+}
+.footer-menu-overlay{
+  background-image: url(./img/Background.jpg);
+  background-repeat: no-repeat;
+  background-position:center;
+  background-repeat: repeat-x;
```
Comment (dmitry-darly-solutions) [inline homework7_8/style.css:896]: duplicate background-repeat

File: homework7_8/index.html [hunk 16]
```diff
@@ -0,0 +1,258 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no"/>
+  <title>homework 7-8</title>
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://allfont.ru/allfont.css?fonts=league-gothic" rel="stylesheet" type="text/css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
```
Comment (dmitry-darly-solutions) [inline homework7_8/index.html]: please download fonts and styles and add to project from local files (https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face for fonts)

File: homework7_8/style.css [hunk 17]
```diff
@@ -0,0 +1,701 @@
...
+  right: 0;
+  transition: right 2s;
+}
+.slider{
+  position: relative;
+  min-width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap:50px;
```
Comment (valeriya-reva) [inline homework7_8/style.css]: gap can be used with display: grid; not with flex

**Pull request merged**